{{cookiecutter.project_slug}}/ci/travis.sh

```diff
@@ -56,19 +54,15 @@ fi
 pip install -U pip setuptools wheel
 
 if [ "$CHECK_FORMATTING" = "1" ]; then
-    pip install yapf==${YAPF_VERSION}
-    if ! yapf -rpd setup.py {{cookiecutter.package_name}}; then
+    pip install black
+    if ! black --diff setup.py {{cookiecutter.package_name}}; then
```

Comment:
Does this work? I think you might need to add `--check` as well, or else `black` always reports success, even if it needed to make changes.

Reply:
yeah, I am updating it